src/ubuntu/18.04/helix/webassembly/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:18.04

```

Comment:
Any reason this can't be based on the Ubuntu 18.04 Helix image instead?

Reply:
done :)